lib/srv/db/proxy_test.go

```diff
@@ -40,17 +40,22 @@ func TestProxyProtocolPostgres(t *testing.T) {
 
 	testCtx.createUserAndRole(ctx, t, "alice", "admin", []string{"postgres"}, []string{"postgres"})
 
-	// Point our proxy to the Teleport's db listener on the multiplexer.
-	proxy, err := multiplexer.NewTestProxy(testCtx.mux.DB().Addr().String())
-	require.NoError(t, err)
-	t.Cleanup(func() { proxy.Close() })
-	go proxy.Serve()
-
-	// Connect to the proxy instead of directly to Postgres listener and make
-	// sure the connection succeeds.
-	psql, err := testCtx.postgresClientWithAddr(ctx, proxy.Address(), "alice", "postgres", "postgres", "postgres")
-	require.NoError(t, err)
-	require.NoError(t, psql.Close(ctx))
+	for _, v2 := range []bool{false, true} {
+		v2 := v2
```

Comment:
Is this necessary?

Reply:
it is now (we pass it down to closure to `NewTestProxy`, I had hardcoded `false` before)

Reply:
But there's no parallel code here, so I think `v2` will be correct on each iteration of the loop.

Reply:
I've added it as a safeguard in case we add something like t.Parallel somewhere down the road. Currently it's true it's not needed. I removed it for now.